committeeoversightapp/models.py

```diff
@@ -9,17 +19,108 @@ class HearingCategoryType(models.Model):
+    category = models.ForeignKey(HearingCategoryType,
+                                 null=True,
+                                 on_delete=models.CASCADE)
+
 
 class WitnessDetails(models.Model):
     witness = models.ForeignKey(EventParticipant, on_delete=models.CASCADE)
-    document = models.ForeignKey(EventDocument, null=True, blank=True, on_delete=models.CASCADE)
-    organization = models.CharField(max_length=100, null=True, blank=True, primary_key=False)
+    document = models.ForeignKey(EventDocument,
+                                 null=True,
+                                 blank=True,
+                                 on_delete=models.CASCADE)
+    organization = models.CharField(max_length=100,
+                                    null=True,
+                                    blank=True,
+                                    primary_key=False)
     retired = models.BooleanField(default=False)
 
+
 class Committee(models.Model):
     lugar_id = models.IntegerField(null=True, blank=True, primary_key=False)
-    lugar_name = models.CharField(max_length=200, null=True, blank=True, primary_key=False)
-    organization = models.ForeignKey(Organization, null=True, blank=True, on_delete=models.CASCADE)
+    lugar_name = models.CharField(max_length=200,
+                                  null=True,
+                                  blank=True,
+                                  primary_key=False)
+    organization = models.ForeignKey(Organization,
+                                     null=True,
+                                     blank=True,
+                                     on_delete=models.CASCADE)
+
+
+class StaticPage(Page):
+    body = StreamField([
+        ('heading', blocks.CharBlock(classname='full title', icon='openquote')),
+        ('paragraph', blocks.RichTextBlock()),
+        ('image', ImageChooserBlock()),
+        ('button',
+         blocks.StructBlock([
+            ('button_text', blocks.CharBlock()),
+            ('button_link', blocks.URLBlock())
+            ],
+         icon='site'))
+    ])
+
+    # Editor configuration
+    content_panels = Page.content_panels + [
+        StreamFieldPanel('body'),
+    ]
+
+
+class LandingPage(Page):
+    body = StreamField([
+        ('heading', blocks.CharBlock(classname='full title', icon='openquote')),
+        ('paragraph', blocks.RichTextBlock()),
+        ('image', ImageChooserBlock()),
+        ('button',
+         blocks.StructBlock([
+            ('button_text', blocks.CharBlock()),
+            ('button_link', blocks.URLBlock())
+            ],
+         icon='site'))
+    ])
+
+    # Editor configuration
+    content_panels = Page.content_panels + [
+        StreamFieldPanel('body'),
+    ]
+
+
+
+class CategoryDetailPage(Page):
+    # model page method adapted from https://timonweb.com/tutorials/how-to-hide-and-auto-populate-title-field-of-a-page-in-wagtail-cms/
+    category = models.ForeignKey(
```

Comment:
right now, it's possible to try and create a new category page for a category that's already in use. you only get an error at the point of trying to save the redundant page, when the slug is autogenerated. it's a bit jarring, but then i may be imagining workflows that no one will actually do irl. do you think any action is needed to prevent creating redundant category pages, e.g., limiting the initial category choices?

Reply:
I'm thinking about that as a nice-to-have, since it would take some time to implement, the site already prevents users from making redundant pages, and we're on a tight deadline. Is there a reason you think it's worth putting time into that now?

Reply:
nope, just wanted to confirm that it's the expected behavior!

Reply:
agree with your rationale for not doing it, fwiw!